### PR TITLE
Hero character count

### PR DESCRIPTION
**Pull request by sebworks**
Hero character count

## Additions

- Added `cfgov/v1/migrations/0060_hero_character_count.py` to add classes to fields.

## Changes

- Modified `cfgov/templates/wagtailadmin/js/hallo-hero-text-count.js` 
- Modified `cfgov/v1/wagtail_hooks.py` to add the hook for the JS file.

## Testing

- Add a Hero to a Sublanding page
- Enter in some text into the heading. Verify that the heading help text is:
  - green when less than 41 characters
  - orange when between 41 and 82 characters
  - red when above 82 characters

- Verify that the body help text is:
   - green when the heading text is less than 41 characters and the body text is less than 165 characters
   - orange when heading text text is less than 41  and the body text is between 165 and 186 characters.
   - red when the heading text is  less than 41 characters and the body text is greater than 186 characters
  - green when the heading text is greater than 41 characters and the body text is less than 108 characters
  - orange when heading text text is greater than 41  and the body text is between 108 and 124 characters.
  - red when the heading text is  greater than 41 characters and the body text is greater than 124 characters.
## Screenshots
- 
screen shot 2017-04-07 at 11 53 19 am

## Notes

-

## Todos

-

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/v1/atomic_elements/molecules.py [hunk 1]
```diff
@@ -68,11 +68,13 @@ class Meta:
 class Hero(blocks.StructBlock):
     heading = blocks.CharBlock(
         required=False,
-        help_text='Maximum character count: 25 (including spaces)'
+        help_text='Maximum character count: 25 (including spaces)',
```
Comment (jimmynotjim) [inline cfgov/v1/atomic_elements/molecules.py]: You're settings say 41 and 82 depending on the situation, is 25 correct here?

Comment (sebworks) [inline cfgov/v1/atomic_elements/molecules.py]: Yep, good point. I'll adjust the copy

File: cfgov/templates/wagtailadmin/js/hallo-hero-text-count.js [hunk 2]
```diff
@@ -0,0 +1,303 @@
+
```
Comment (sebworks) [inline cfgov/templates/wagtailadmin/js/hallo-hero-text-count.js]: yeah, how do handle the login though?

**Comment by sebworks** [comment]
This is on hold until we get the Wagtail testing figured out.

**Comment by sebworks** [comment]
I'm closing, I don't think it's needed at the current time.

**Pull request closed without merging**